maven-surefire-plugin/src/site/apt/examples/fork-options-and-parallel-execution.apt.vm

```diff
@@ -181,6 +181,67 @@ Fork Options and Parallel Test Execution
   See the keywords: <volatile>, <synchronized>, <<immutable>> and <final> in
   {{{https://jcp.org/en/jsr/detail?id=133}Java Memory Model - JSR-133}}.
 
+* Parallel Test-Suite Execution
+
+  ${project.artifactId}'s notion of "suite" is related to
+  {{{https://github.com/junit-team/junit4/wiki/Aggregating-tests-in-suites}junit4 Suite}}. 
+
+  For example, say your tests are laid out like this:
+
++---+
+src/test/java
++-features.areaA
+| +-SomeStory.java
+| +-AnotherStory.java
+| ...
++-features.areaB
+| +-DifferentStory.java
+| +-OtherStory.java
+| ...
+...
++---+
+
+  You would add a TestSuite.java for each package, that would look something like:
+
++---+
+package features.areaA;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+    SomeStory.class,
+    AnotherStory.class,
+    ...
+})
+public class TestSuite {
+    /* Suite holder */
+}
++---+
+
+  and similarly for <<<package features.areaB>>>. You would then configure
+  ${project.artifactId} like so:
+
++---+
+<plugin>
+  <groupId>${project.groupId}</groupId>
+  <artifactId>${project.artifactId}</artifactId>
+  <version>${project.version}</version>
+  <configuration>
+    <includes>
+      <include>features.*.*Suite</include>
```

Comment:
Here it should be `<include>features/*/TestSuite.java</include>`.
We have only one file name `TestSuite.java` in two directories, right?

Reply:
Could go either way.

- First, I am using [fully-qualified name](https://maven.apache.org/surefire/maven-failsafe-plugin/examples/inclusion-exclusion.html#Fully_qualified_class_name); although that feature has [issues](https://issues.apache.org/jira/browse/SUREFIRE-1789).
- Second, I do not see a problem with the wildcard `*Suite`, but `TestSuite` would work as well in this case.

Reply:
If you use `TestSuite` the users would understand more what you mean. And the same way you declare the rule of your feature that this is the root of the tree where all the children - suites continue right after.

Reply:
sounds good